sqlglot/__init__.py

```diff
@@ -27,7 +29,7 @@ def parse(code, read=None, **opts):
     return dialect.parse(code, **opts)
 
 
-def parse_one(code, read=None, **opts):
+def parse_one(code, read=None, **opts) -> Expression:
```

Comment:
if you're going to do this might as well be consistent and put on line 49 -> str...